workspaces/topology/plugins/topology/package.json

```diff
@@ -0,0 +1,112 @@
+    "supported-versions": "1.28.4",
+    "pluginId": "topology",
+    "pluginPackages": [
+      "@backstage-community/plugin-topology",
+      "@backstage-community/plugin-topology-common"
+    ]
+  },
+  "sideEffects": [
+    "./**/*.css"
+  ],
+  "scripts": {
+    "build": "backstage-cli package build",
+    "clean": "backstage-cli package clean",
+    "export-dynamic": "janus-cli package export-dynamic-plugin --in-place",
+    "lint": "backstage-cli package lint",
+    "postpack": "backstage-cli package postpack",
+    "postversion": "yarn run export-dynamic",
+    "prepack": "backstage-cli package prepack",
+    "start": "backstage-cli package start",
+    "test": "backstage-cli package test --passWithNoTests --coverage",
+    "tsc": "tsc"
+  },
+  "dependencies": {
+    "@backstage-community/plugin-topology-common": "1.4.0",
+    "@backstage/catalog-model": "^1.5.0",
+    "@backstage/core-components": "^0.14.9",
+    "@backstage/core-plugin-api": "^1.9.3",
+    "@backstage/plugin-catalog-react": "^1.12.2",
+    "@backstage/plugin-kubernetes": "^0.11.12",
+    "@backstage/plugin-kubernetes-common": "^0.8.1",
+    "@backstage/plugin-kubernetes-react": "^0.4.1",
+    "@backstage/plugin-permission-react": "^0.4.24",
+    "@backstage/theme": "^0.5.6",
+    "@janus-idp/shared-react": "2.11.1",
+    "@kubernetes/client-node": "^0.20.0",
+    "@material-ui/core": "^4.9.13",
+    "@material-ui/icons": "^4.11.3",
+    "@material-ui/lab": "^4.0.0-alpha.45",
+    "@mui/icons-material": "5.15.17",
+    "@patternfly/patternfly": "^5.1.0",
+    "@patternfly/react-charts": "^7.1.1",
+    "@patternfly/react-core": "^5.1.1",
+    "@patternfly/react-styles": "^5.1.1",
+    "@patternfly/react-tokens": "^5.1.1",
+    "@patternfly/react-topology": "^5.1.0",
+    "classnames": "2.x",
+    "git-url-parse": "^13.1.0",
+    "js-yaml": "^4.1.0",
+    "lodash": "^4.17.21",
+    "react-use": "^17.4.0"
+  },
+  "peerDependencies": {
+    "react": "^16.13.1 || ^17.0.0 || ^18.0.0",
+    "react-dom": "^16.13.1 || ^17.0.0 || ^18.0.0",
+    "react-router-dom": "^6.0.0"
+  },
+  "devDependencies": {
+    "@backstage/cli": "0.26.11",
+    "@backstage/core-app-api": "1.14.1",
+    "@backstage/dev-utils": "1.0.36",
+    "@backstage/test-utils": "1.5.9",
+    "@janus-idp/cli": "1.15.1",
+    "@redhat-developer/red-hat-developer-hub-theme": "0.3.0",
+    "@testing-library/jest-dom": "6.4.8",
+    "@testing-library/react": "14.3.1",
+    "@testing-library/react-hooks": "8.0.1",
+    "@testing-library/user-event": "14.5.2",
+    "@types/git-url-parse": "9.0.3",
+    "@types/node": "18.19.34",
+    "@types/react": "^18.3.11",
+    "cross-fetch": "4.0.0",
+    "msw": "1.3.3",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
+    "react-router-dom": "^6.26.2"
+  },
+  "files": [
+    "dist",
+    "dist-scalprum",
+    "app-config.janus-idp.yaml"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/janus-idp/backstage-plugins",
```

Comment:
Can you change the url here to refer to the community-plugins repo? Please change the issues url as well